### ISSUE TITLE
No player status

### ISSUE THREAD

**Issue by kadenstratton** [NamelessMC/Nameless#997]
The widgets on homepage and play page that show how many people on the server are not working. All commands are working in game and are connected with the website. My domain is skypulse.us

**Comment by samerton** [comment]
The status on the website comes from a website query instead of the plugin in version 1.

You can *try* enabling the external query in the AdminCP -> Minecraft tab to see if that helps.

**Comment by kadenstratton** [comment]
Now it shows the website is online but not how many people are in. Also once a user registers through game on my website, what do they use as their password?

**Comment by samerton** [comment]
Unfortunately that's the problem with the external query, it does not retrieve a player list in version 1.
If you are using a web host, you can try asking them if port 25565 (or whichever port your Minecraft server runs on) is unblocked, and also if the use of sockets is enabled (the fsockopen function specifically), as these are requirements to use the internal query.

When a user registers ingame, Nameless will attempt to send them an email with instructions on how to set their password. If they can't find it in their inbox, they can try searching their junk/spam folder to see if it's ended up in there, and if they haven't received it at all, you can find out more information on the issue [here](https://github.com/NamelessMC/Nameless/wiki/FAQs#nameless-wont-send-emails-to-users).

**Comment by kadenstratton** [comment]
Okay, thanks! How would I go about making a site wide background image. I have asked you this before a year or two ago but I am unable to find the post regarding that.

**Comment by kadenstratton** [comment]
Also, when I am on my donate page and vote page the account buttons, notifications and messaging don't work.

**Comment by samerton** [comment]
To set a background image for your site, you can follow the steps at the bottom of [this post](https://www.spigotmc.org/threads/nameless-minecraft-website-software.34810/page-147#post-1603646).

As for the dropdown issues on the donate and vote pages, you'll first need to configure each addon in the AdminCP -> Addons tab by clicking the settings icon next to them.

**Comment by samerton** [comment]
Were you able to resolve the issue @kadenstratton?

**Comment by kadenstratton** [comment]
Yep! I just tried it. I haven't had much time this week to work on it. Thank you!

**Comment by samerton** [comment]
Great, no problem :)

I'll go ahead and mark this as closed, feel free to re-open/open a new issue if you have any more questions

**Comment by kadenstratton** [comment]
The background image is not working for me. This is what I am adding to custom.css in the current theme.

body {
    background-image: url('/skypulse.us/background.png');
    background-repeat: no-repeat;
    background-attachment: fixed;
    background-size: cover;
}

**Comment by samerton** [comment]
@kadenstratton 
The path needs to be relative to the URL, for example try changing it to just `background-image: url('/background.png');`

**Comment by kadenstratton** [comment]
It worked! Thank you!
